Toric/Mathlib/CategoryTheory/WithTerminal/Cones.lean

```diff
@@ -0,0 +1,137 @@
+import Mathlib.CategoryTheory.Comma.Over.Basic
+import Toric.Mathlib.CategoryTheory.WithTerminal
+
+/-!
+# Relations between `Cone`, `WithTerminal` and `Over`
+
+Given a categories `C` and `J`, an object `X : C` and a functor `K : J ⥤ Over X`,
+it has an obvious lift `liftFromOver K : WithFinal J ⥤ C`. These two functors have
+equivalent categories of cones (`coneEquiv`). As a corollary, the limit of `K` "is" the
+limit of `liftFromOver K`, and viceversa
+-/
+
+open CategoryTheory CategoryTheory.Limits CategoryTheory.WithTerminal
+
+universe w w' v₁ v₂ u₁ u₂
+variable {C : Type u₁} [Category.{v₁} C]
+variable {D : Type u₂} [Category.{v₂} D]
+variable {J : Type w} [Category.{w'} J]
+
+namespace CategoryTheory.Limits.WithTerminal
+
+@[simps]
+def asNatTransf {X : C} (K : J ⥤ Over X) :
```

Comment:
You are missing a lot of docstrings on definitions (anything you declare with `def`). Could you document them?.

A linter should be catching this, but clearly it's not doing its job 🤔